### ISSUE TITLE
No images shown for mp3

### ISSUE THREAD

**Issue by quelbs** [mkulesh/onpc#40]
For me no images of the mp3 are shown, I only see the empty_cover.jpg image (the integra app shows the images).
My device is a TX-8150

**Comment by quelbs** [comment]
Debug states the image is loaded, but cannot be opened:
```
D/onpc: JacketArtMsg: Cover image size length=201774
D/skia: --- SkAndroidCodec::NewFromStream returned null
D/onpc: JacketArtMsg: can not open image
```

**Comment by mkulesh** [comment]
Hi @quelbs, I found the problem. The RAW image stream that is available at http://192.168.1.80/album_art.cgi contains a header like this one

`Content-size: 17972
Content-type: image/jpeg`

This header has for my player the Linux encoding, and for your receiver - the Windows encoding. Of course I have newer consider Windows encoding here :) 

Now it is fixed and I hope all images shall be shown for you.

**Comment by quelbs** [comment]
Great, works fine for me. Thank you very much!

**Comment by mkulesh** [comment]
Hi @quelbs, that do you mean with "switch apps and switch back..."?

**Comment by quelbs** [comment]
- minimize app (press home button) an open app again > album art is not displayed
- open settings, go back > album art is not displayed

I will provide logs tonight

**Comment by mkulesh** [comment]
Hi @quelbs, yes, I need logging since this cases work for me as desired; i.e the album art is always updated.

**Comment by quelbs** [comment]
Logs are on their way.

**Comment by quelbs** [comment]
The problem with covers not showing still occurs when opening the app after it has been closed or another app has been active. After changing a setting the cover is displayed again now.
Also on track change while in LISTEN tab the cover of the new track is still not shown for me.

Maybe it would be nice to have an indicator that the app currently is loading the cover, because currently there is no difference between "no cover available, thus displaying the empty cover" and "cover is currently loading".

**Comment by mkulesh** [comment]
Hi @quelbs 

> The problem with covers not showing still occurs when opening the app after it has been closed or another app has been active.

Really strange behavior. I think I need logging that contains app restart.

> Maybe it would be nice to have an indicator that the app currently is loading the cover, because currently there is no difference between "no cover available, thus displaying the empty cover" and "cover is currently loading".

Devices like my player force to update cover image 2 or 3 times if track is changed. If implementig this proposal, the cover image will flip-flop at track change. However, the indication of sand glass in the LISTEN tab above the cover image is possible for this operation. That do you think about a sand glass?

**Comment by quelbs** [comment]
I will try to provide logs soon.
Regarding one indicator: an overlay (like an hourglass) was what I thought of.

**Comment by quelbs** [comment]
It looks like b81e182 fixed this issue for me. I did not find any problem and the intermediate problem of d03ff54 that album art was shown although no album art was present (e.g. for DAB stations) is also fixed.

**Comment by mkulesh** [comment]
Released in version 1.0
